backend/src/main/java/de/sambam/gamenighttracker/db/GameDb.java

```diff
@@ -0,0 +1,35 @@
+package de.sambam.gamenighttracker.db;
+
+import de.sambam.gamenighttracker.model.Game;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class GameDb {
+    private final List<Game> gameList;
+
+    public GameDb() {
+        this.gameList = new ArrayList<>(List.of(
+                Game.builder().name("Cubitos").releaseYear("2020").build(),
+                Game.builder().name("CloudAge").releaseYear("2020").build(),
+                Game.builder().name("Barrage").releaseYear("2019").build()
+        ));
+    }
+
```

Comment:
```suggestion
    private final List<Game> gameList =  new ArrayList<>(List.of(
                Game.builder().name("Cubitos").releaseYear("2020").build(),
                Game.builder().name("CloudAge").releaseYear("2020").build(),
                Game.builder().name("Barrage").releaseYear("2019").build()
        ));
```
you could also initialize the variable directly